galasa-managers-parent/galasa-managers-core-parent/dev.galasa.textscan.manager/src/test/resources/loremIpsum.txt

```diff
@@ -0,0 +1,98 @@
+Lorem ipsum dolor sit amet, consectetur adipiscing elit. 
```

Comment:
Ok,  this isn't working,   I can't validate the junit tests.   Would you please simplify the input text with stuff like
Line 1
Line 2
Line 3 Match 1
etc